<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_interviewee</class>
 <widget class="QDialog" name="Dialog_interviewee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>160</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="student_id"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>专业</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="major">
        <item>
         <property name="text">
          <string>计算机科学与技术</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>软件工程</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>信息安全</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>数字媒体技术</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>工业设计</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>产业设计</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>专业班级</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="class_num"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phone"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>籍贯</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="birth_place"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>160</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>生日</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="birthday"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>兴趣爱好</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="habit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>个人简介</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="introduction"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>第一志愿</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="first">
        <item>
         <property name="text">
          <string>办公室</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>人资部门</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>组织部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宣传部</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>第二志愿</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="second">
        <item>
         <property name="text">
          <string>办公室</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>人资部门</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>组织部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宣传部</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>是否接受调剂</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="is_subjective">
        <item>
         <property name="text">
          <string>是</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>否</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="register_button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>510</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>面试者信息查询</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>方正剪纸繁体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>面试者信息查询</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>133</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>方正剪纸繁体</family>
      </font>
     </property>
     <property name="text">
      <string>请输入学号进行查询</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stu_id_line">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>130</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search_information_button">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>129</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="session_table">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>517</width>
       <height>56</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>面试场次</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>面试时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>面试地点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>通过状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>面试分数</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
